instance/tests/models/test_openedx_storage_mixins.py

```diff
@@ -21,18 +21,22 @@
 """
 
 # Imports #####################################################################
-from unittest.mock import patch, call
+from unittest.mock import call, patch
 
 import boto
 import ddt
 import yaml
 from django.conf import settings
 from django.test.utils import override_settings
 
-from instance.models.mixins.storage import get_s3_cors_config, get_master_iam_connection, StorageContainer
+from instance.models.mixins.storage import (StorageContainer,
+                                            get_master_iam_connection,
+                                            get_s3_cors_config)
 from instance.tests.base import TestCase
-from instance.tests.models.factories.openedx_appserver import make_test_appserver
-from instance.tests.models.factories.openedx_instance import OpenEdXInstanceFactory
+from instance.tests.models.factories.openedx_appserver import \
```

Comment:
edX uses 120 as line length so you may want to set that in your IDE and undo these formatting changes.